src/theme/index.tsx

```diff
@@ -21,6 +21,7 @@ export const brand = brandLight
 
 const stateColors = {
   active: 'hsl(221, 96%, 64%)',
+  activeSoft: transparentize(0.76, 'hsl(221, 96%, 64%)'),
```

Comment:
wait similar comment as zach here, can this be done with an alpha value in the `hsl` or do u need transparentize